reference/configuration/security.rst

```diff
@@ -61,6 +62,19 @@ If ``true``, the user is asked to authenticate before each call to the
 ``isGranted()`` method in services and controllers or ``is_granted()`` from
 templates.
 
+anonymous
+~~~~~~~~~
+
+**type**: ``string`` **default**: ``~``
+
+When set to ``lazy``, Symfony loads the user (and starts the session) only if the
+application actually accesses the `User` object (e.g. via a `is_granted()` call in
```

Comment:
we need double backticks hier and below when enclosing code

Reply:
Done :-)